src/toolkits/coreml_export/neural_net_models_exporter.cpp

```diff
@@ -390,33 +390,31 @@ std::shared_ptr<MLModelWrapper> export_activity_classifier_model(
 
 std::shared_ptr<coreml::MLModelWrapper> export_style_transfer_model(
     const neural_net::model_spec& nn_spec, size_t image_width,
-    size_t image_height, flex_dict user_defined_metadata) {
+    size_t image_height, bool include_flexible_shape,
+    flex_dict user_defined_metadata) {
   CoreML::Specification::Model model;
   model.set_specificationversion(3);
 
   ModelDescription* model_desc = model.mutable_description();
 
-  ImageFeatureType* input_feat =
-      set_image_feature(model_desc->add_input(), image_width, image_height,
-                        "image", "Input image");
-
-  /**
-   * The -1 indicates no upper limits for the image size
-   */
-  set_image_feature_size_range(input_feat, 64, -1, 64, -1);
+  FeatureDescription* model_input = model_desc->add_input();
+  ImageFeatureType* input_feat = set_image_feature(model_input, image_width, image_height, "Input image");
+  model_input->set_name("image");
```

Comment:
Why can't we set the name "image" as part of the call to `set_image_feature`?

Reply:
At the time I coded it the set_image_feature didn't have an option for an image name and there were too many conflicts to resolve so I thought this was the best solution. Now that implementation has changed so I'll refactor to use `set_image_feature`.